jbpm-wb-process-instance-migration/pim-service/src/main/java/org/kie/processmigration/gui/model/RunningInstance.java

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright 2018 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.kie.processmigration.gui.model;
+
+import java.util.Date;
+import org.kie.server.api.model.instance.ProcessInstance;
+
+import java.text.SimpleDateFormat;
+import javax.xml.datatype.XMLGregorianCalendar;
+
+public class RunningInstance {
+
+    private int id;
```

Comment:
why do we need int id? remove this and use processInstanceId instead

Reply:
That is because this id is used in the React PageMigrationRunningInstances's pagination. Note processInstanceId might not be from 1 to N, it could be 2, 5, 9, etc, thus processInstanceId can't be used in pagination's calculation.

Reply:
What exactly is posing such limitation?